lib/bower-plugin.js

```diff
@@ -53,19 +54,20 @@ function BowerWebpackPlugin(options) {
   this.manifestFiles = opt.manifestFiles ? [].concat(opt.manifestFiles) : ["bower.json"];
   this.includes = opt.includes ? [].concat(opt.includes) : [/.*/];
   this.excludes = opt.excludes ? [].concat(opt.excludes) : [];
+  this.searchResolveModulesDirectories = opt.searchResolveModulesDirectories === false ? false : true;
```

Comment:
Er whoops it's actually `true` by default. Unless the user explicitly sets `false` in the `options`, is will search through `resolve.modulesDirectories`.